configs.js

```diff
@@ -0,0 +1,42 @@
+let alert = document.getElementById('alert');
+
+function showAlert(){
+       alert.style.display = 'block'
+    setTimeout(function () {
+        alert.style.display = 'none'
+    }, 20000)
+}
+
+function restoreOptions() {
+    chrome.storage.sync.get(
+        'mode',
+        function (items) {
+            if (items.mode === 'enabled') {
+                document.getElementById('toggleBtn').innerText = 'ON';
+            } else {
+                document.getElementById('toggleBtn').innerText = 'OFF';
+            }
+        });
+}
+
+function reloadPage(){
+    chrome.tabs.reload();
+}
+
+document.getElementById('toggleBtn').addEventListener('click', function () {
+    if (this.innerText == 'ON') {
```

Comment:
Can we make this toggle based on the saved value (from Chrome storage) rather than the text value?